Messaging.md

```diff
@@ -359,35 +362,77 @@ Note that `outcome_i` is the outcome value itself and not its hash that will be
    * [`int32`:`precision`]
    * [`u16`:`nb_digits`]
 
+### The `oracle_event_timestamp` Type
+
+This type contains timestamp information about when an `oracle_event` is to happen.
+
+#### Fixed `oracle_event_timestamp`
+
+1. type: 55348 (`oracle_event_timestamp_fixed`)
+2. data:
+   * [`u32`:`expected_time_epoch`]
+
+#### Range `oracle_event_timestamp`
+
+1. type: 55350 (`oracle_event_timestamp_range`)
+2. data:
+   * [`u32`:`earliest_expected_time_epoch`]
+   * [`u32`:`latest_expected_time_epoch`]
+
 ### The `oracle_event` Type
 
 This type contains information provided by an oracle on an event that it will attest to.
 See [the Oracle specifications](./Oracle.md#oracle-event) for more details.
 
-#### Version 0 `oracle_event`
+#### Version 1 `oracle_event`
+
+1. type: 55352 (`oracle_event`)
 
-1. type: 55330 (`oracle_event_v0`)
 2. data:
+
    * [`u16`:`nb_nonces`]
    * [`nb_nonces*x_point`:`oracle_nonces`]
-   * [`u32`:`event_maturity_epoch`]
+
+   * [`oracle_event_timestamp`:`timestamp`]
    * [`event_descriptor`:`event_descriptor`]
    * [`string`:`event_id`]
 
+### The `oracle_keys` Type
+
+This type contains static oracle information and can be used to import trusted oracles into wallets.
```

Comment:
Do you mean static in the sense of fixed-for-this-outcome-announcement-lifecycle ? If so add requirement that keys should be saved in consequence by the DLC node.